src/userProfileService.ts

```diff
@@ -1,163 +1,76 @@
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-import request, { Method } from 'axios';
+// Modules
+import Axios from 'axios';
+
 // @ts-ignore
 import { getAccessToken } from './authService';
 
-interface User {
-  preferredService?: string | null;
-  profilePictureUrl?: string | null;
-  profilePhotoUrl?: string | null;
-  churchEntityKnown?: boolean | null;
-  churchEntityId?: number | null;
-  churchEntityName?: string | null;
-  prefix?: string | null;
-  middleName?: string | null;
-  suffix?: string | null;
-  nickName?: string | null;
-  fullName?: string | null;
-  gender?: string | null;
-  maritalStatusId?: number | null;
-  maritalStatus?: string | null;
-  membershipStatusId?: number | null;
-  membershipStatus?: string | null;
-  disengagementReason?: string | null;
-  birthDate?: string | null;
-  deceasedDate?: string | null;
-  age?: number | null;
-  allergies?: string | null;
-  gradeLevel?: string | null;
-  departmentId?: number | null;
-  departmentName?: string | null;
-  departmentChurchEntityId?: number | null;
-  departmentChurchEntityName?: string | null;
-  preferredServiceEventId?: number | null;
-  modifiedDate?: number | null;
-  isAdult?: boolean | null;
-  isChild?: boolean | null;
-  isStudent?: boolean | null;
-  milestones?: Array<any> | null;
-  addresses?: Array<any> | null;
-  emails?: Array<any> | null;
-  phones?: Array<any> | null;
-  occupations?: Array<any> | null;
-  contactPreferences?: Object | null;
-  emergencyContacts?: Array<any> | null;
-  id: number | null;
-  firstName: string | null;
-  lastName: string | null;
-}
-
-export const getUserProfile = async (url: string) => {
-  let user: any = null;
-  try {
-    const accessToken = await getAccessToken();
-
-    if (!accessToken) {
-      return null;
-    }
-
-    const options = {
-      method: 'GET' as Method,
-      headers: { Authorization: `Bearer ${accessToken}` },
-      url,
-    };
-
-    const { data } = await request(options);
-    user = data;
-    return user;
-  } catch (error) {
-    return null;
-  }
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { IUser } from './interfaces/user.interface';
```

Comment:
@ajaybushel we need to talk about our convention here make sure we are following the same on all repos including the mobile app. I will setup a meeting to sync up on that